<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriParaEdit</class>
 <widget class="QDialog" name="TriParaEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>position</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>material</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox"/>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="xLabel_2">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="xDoubleSpinBox_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>vertex</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="xLabel_3">
          <property name="text">
           <string>z</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="xDoubleSpinBox_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboBox_3"/>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="comboBox_4"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TriParaEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TriParaEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
